tests/layer_tests/ovc_python_api_tests/test_onnx.py

```diff
@@ -115,11 +115,16 @@ def test_unicode_paths(self):
 
             assert os.path.exists(model_path), "Could not create a directory with unicode path."
 
-            from openvino import convert_model
+            from openvino import convert_model, save_model, Core
             res_model = convert_model(model_path)
             flag, msg = compare_functions(res_model, model_ref, False)
             assert flag, msg
 
+            save_model(res_model, model_path + ".xml")
+            res_model_after_saving = Core().read_model(model_path + ".xml")
+            flag, msg = compare_functions(res_model_after_saving, model_ref, False)
```

Comment:
how do we test here that Unicode paths work well?
I think we need explicit tests for it

Reply:
In this test the temporary directory is created with unicode symbols: https://github.com/openvinotoolkit/openvino/blob/214c5e87bef34657ed625fd2e481520a7ed16723/tests/layer_tests/ovc_python_api_tests/test_onnx.py#L106

If we try to save the model in such directory on master, the model is saved in the corrupted path like: `C:\...\ÐœÐ¾Ð´ÐµÐ»ÑŒ\model.xml `

So in the test I save the model with `save_model()` than read the model from the same path with `read_model()` (`read_model` works fine with unicode on master) than check that the model is correct.

On master this test fails as `read_model()` can't find the model in directory.